arrow-udf-deno/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "arrow-udf-deno"
```

Comment:
How about renaming this crate to `arrow-udf-js-deno`? Because from users' view, they write UDFs in JavaScript language. Deno is just one of the runtimes of the JavaScript.

Reply:
Sure, no problem